resources/qml/ApplicationSwitcher/ApplicationSwitcherPopup.qml

```diff
@@ -57,10 +57,10 @@ Popup
                     permissionsRequired: []
                 },
                 {
-                    displayName: "UltiMaker Academy", //Not translated, since it's a brand name.
-                    thumbnail: UM.Theme.getIcon("Knowledge"),
-                    description: catalog.i18nc("@tooltip:button", "Become a 3D printing expert with UltiMaker e-learning."),
-                    link: "https://academy.ultimaker.com/?utm_source=cura&utm_medium=software&utm_campaign=switcher-academy",
+                    displayName: "Sponsor Cura", //Not translated, since it's a brand name.
```

Comment:
I would say we would translate this since it is not a brand anymore
```suggestion
                    displayName: catalog.i18nc("@label:button", "Sponsor Cura"),
```